cumulus_library/studies/core/builder_soe.py

```diff
@@ -0,0 +1,57 @@
+""" Module for generating condition codeableConcept table"""
+import re
+from cumulus_library.base_table_builder import BaseTableBuilder
+from cumulus_library.helper import get_progress_bar, query_console_output
+from cumulus_library.template_sql.templates import (
+    get_column_datatype_query,
+    get_object_denormalize_query,
+)
+
+
+class SOEBuilder(BaseTableBuilder):
+    display_text = "Creating SoE support tables..."
+
+    def prepare_queries(self, cursor: object, schema: str):
+        """Constructs tables for SOE QA verification
+
+        :param cursor: A database cursor object
+        :param schema: the schema/db name, matching the cursor
+
+        """
+        table = "documentreference"
+        column = "context"
+        with get_progress_bar(transient=True) as progress:
```

Comment:
reminder: did you want to squeeze in the time-elapsed instead of time-remaining change?

Reply:
I want to leave that as its own thing rather that tie it to another feature.